src/functions/action-status.js

```diff
@@ -54,10 +54,10 @@ export async function actionStatus(
     content: reaction
   })
 
-  // remove the initial reaction on the IssueOp comment that triggered this action
-  await octokit.rest.reactions.deleteForIssueComment({
-    ...context.repo,
-    comment_id: context.payload.comment.id,
-    reaction_id: reactionId
-  })
+  // // remove the initial reaction on the IssueOp comment that triggered this action
+  // await octokit.rest.reactions.deleteForIssueComment({
+  //   ...context.repo,
+  //   comment_id: context.payload.comment.id,
+  //   reaction_id: reactionId
+  // })
```

Comment:
Im not sure why, but changing the way the locks worked broke this. All this is doing is removing an emoji to the comment. So its not super important but might want to figure out before we move forward.